db/fixtures/miq_shortcuts.yml

```diff
@@ -386,7 +356,7 @@
   :startup: true
 - :name: ems_object_storage
   :description: Storage / Object Storage / Managers
-  :url: /ems_object_storage/show_list
+  :url: /ems_storage/show_list
```

Comment:
I'm surprised these 2 would go to the same location. I thought we had dedicated endpoints for these. @agrare Do you know?

Reply:
We combined "Object" and "Block" storage managers into a single section https://github.com/ManageIQ/manageiq-ui-classic/pull/7746